src/types/DefaultValue.ts

```diff
@@ -0,0 +1,25 @@
+import { ErrorMessage } from './ErrorMessage';
+import { Status } from './Status';
+import { Todo } from './Todo';
+
+export interface DefaultValue {
+  todos: Todo[];
+  setTodos: (value: Todo[] | ((value: Todo[]) => Todo[])) => void;
```

Comment:
You can type your setter using the default type it has:
`React.Dispatch<React.SetStateAction<Todo[]>>`